rma/views/rma_views.xml

```diff
@@ -0,0 +1,269 @@
+                            <field name="delivery_picking_count" widget="statinfo" string="Delivery"/>
+                        </button>
+                        <button type="object"
+                                string="Refund"
+                                name="action_view_refund"
+                                class="oe_stat_button"
+                                icon="fa-pencil-square-o"
+                                attrs="{'invisible': [('refund_id', '=', False)]}">
+                        </button>
+                    </div>
+                    <div class="oe_title">
+                        <h1>
+                            <field name="name" readonly="1"/>
+                        </h1>
+                    </div>
+                    <group>
+                        <group>
+                            <field name="partner_id" widget="res_partner_many2one" context="{'search_default_customer':1, 'show_address': 1, 'show_vat': True}" options="{'always_reload': True}"/>
+                            <field name="partner_invoice_id"/>
+                            <field name="picking_id"
+                                   options="{'no_create': True}"/>
+                            <field name="move_id"
+                                   attrs="{'required': [('picking_id', '!=', False)]}"
+                                   options="{'no_create': True}"/>
+                            <field name="product_id"
+                                   force_save="1"
+                                   attrs="{'readonly': ['|', ('picking_id', '!=', False), ('state', '!=', 'draft')]}"/>
+                            <label for="product_uom_qty"/>
+                            <div class="o_row">
+                                <field name="product_uom_qty"/>
+                                <field name="product_uom"
+                                       groups="uom.group_uom"/>
+                            </div>
+                            <field name="delivered_qty"
+                                   attrs="{'invisible': [('delivered_qty', '=', 0.0)]}"/>
+                        </group>
+                        <group>
+                            <field name="date"/>
+                            <field name="user_id"/>
+                            <field name="team_id"/>
+                            <field name="origin"/>
+                            <field name="company_id"
+                                   options="{'no_create': True}"
+                                   groups="base.group_multi_company"/>
+                        </group>
+                    </group>
+                    <notebook>
+                        <page name="page_other"
+                              string="Other Information">
+                            <group>
+                                <group>
+                                    <field name="procurement_group_id"/>
```

Comment:
I think you should explain a little bit as it is a little technical for user. Or maybe rename the label ?